### PR TITLE
Update docs with create schema

### PR DESCRIPTION
**Pull request by LavMatt**
adding in documentation for `/data-product/{data-product-name}/table/{table-name}/schema` endpoint

### CODE REVIEW

File: containers/daap-docs/src/var/task/swagger.json [hunk 1]
```diff
@@ -228,10 +228,119 @@
           }
         }
       }
+    },
+    "/data-product/{data-product-name}/table/{table-name}/schema": {
+      "post": {
+        "tags": [
+          "schema"
+        ],
+        "summary": "Register a schema (blue print of your table) for a new table in your Data Product that you want to upload to the Data Platform.",
```
Comment (MatMoore) [inline containers/daap-docs/src/var/task/swagger.json]:
```suggestion
        "summary": "Register a schema (blueprint of your table) for a new table in your Data Product that you want to upload to the Data Platform.",
```

File: containers/daap-docs/src/var/task/swagger.json [hunk 2]
```diff
@@ -228,10 +228,119 @@
...
+        "description": "Registers the first version of your Data Product (by submitting it's metadata)  and enables you to proceed with uploading data.",
```
Comment (MatMoore) [inline containers/daap-docs/src/var/task/swagger.json]: I think this description needs updating - this looks like the description of the /register endpoint?

File: containers/daap-docs/src/var/task/swagger.json [hunk 3]
```diff
@@ -244,7 +353,10 @@
         }
       },
       "dataProductMetadataSchema": {
-        "$ref": "https://raw.githubusercontent.com/ministryofjustice/modernisation-platform-environments/main/terraform/environments/data-platform/data-product-metadata-json-schema/v1.0.0/moj_data_product_metadata_spec.json"
+        "$ref": "https://raw.githubusercontent.com/ministryofjustice/modernisation-platform-environments/main/terraform/environments/data-platform/data-product-metadata-json-schema/v1.1.0/moj_data_product_metadata_spec.json"
+      },
+      "dataProductSchemaSchema": {
```
Comment (MatMoore) [inline containers/daap-docs/src/var/task/swagger.json:358]: is this schema describing the full glue schema, or the stripped down schema we are using for the API?

Comment (LavMatt) [reply containers/daap-docs/src/var/task/swagger.json:358]: stripped down version

File: containers/daap-docs/src/var/task/swagger.json (hunk 2 above)
Comment (LavMatt) [reply containers/daap-docs/src/var/task/swagger.json]: yep have updated thanks

**Review by MatMoore — APPROVED** [review]

**Pull request merged**